src/xdp.props

```diff
@@ -23,6 +23,8 @@
   </PropertyGroup>
   <ItemGroup>
     <!-- eBPF currently has separate packages for each architecture. -->
+    <PackageReference Include="Microsoft.Windows.SDK.CPP.$(Platform)" Version="10.0.26100.1591" />
+    <PackageReference Include="Microsoft.Windows.WDK.$(Platform)" Version="10.0.26100.1591" /> 
```

Comment:
I don't know how this will play with OneBranch builds, which inject their own kits/packages from the Windows OS branch.

Reply:
Yes, this would have to be conditional if we were to go down this route.